### PR TITLE
Mikibo save geojson for destination

### PR DESCRIPTION
**Pull request by mikib0**
Save geojson to db directly instead of modelling osm types. This will simplify lots of things and fix lots of the issues faced with saving osm while also preventing duplication.

### CODE REVIEW

**Comment by andrew-bierman** [comment]
Thanks for this, but looks like we are missing the geojson model you referenced. Did you intend to have a model for this, or just include geojson  in one of the other schemas?

File: server/controllers/tripController.js [hunk 1]
```diff
@@ -1,7 +1,5 @@
 import Trip from "../models/tripModel.js";
-import Node from "../models/osm/nodeModel.js";
-import Way from "../models/osm/wayModel.js";
-import Relation from "../models/osm/relationModel.js";
+import GeoJSON from "../models/geojsonModel.js";
```
Comment (andrew-bierman) [inline server/controllers/tripController.js:2]: missing file

**Review by andrew-bierman — CHANGES_REQUESTED** [review]

File: server/controllers/tripController.js (hunk 1 above)
Comment (mikib0) [reply server/controllers/tripController.js:2]: forgot to include it in the commit 😅

**Review by andrew-bierman — APPROVED** [review]
nicely done

**Pull request merged**